projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/SampleCode.java

```diff
@@ -1,10 +1,50 @@
 package life.qbic.projectmanagement.domain.project.sample;
 
+import java.io.Serial;
+import java.io.Serializable;
 import java.util.Objects;
 
-public record SampleCode(String code) {
+/**
+ * The sample code of a sample.
+ * <li> Must not be empty or null
```

Comment:
Then it should be checked during construction with an IllegalArgumentException. Else, the requirement can be removed.